version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.12.0"
+ThisBuild / version := "0.12.1"
```

Comment:
this should be 0.13.0 to be consistent with the json-databind update

Reply:
@realstraw oh right, bumping

Reply:
@realstraw done